client/src/pages/UpdateGoal.tsx

```diff
@@ -178,10 +214,19 @@ function UpdateGoal() {
             </P>
           </GoalInfoWrapper>
           <NotesWrapper>
-            <PageSlider label="Pages Read" max={numPagesTotal} />
+            <PageSlider
+              label="Pages Read"
```

Comment:
need to add `min={pagesRead}` - that way the slider and input start at the number of pages read so far and not at 0 every time we update